extraConfigSriov.py

```diff
@@ -156,14 +164,13 @@ def run(self, _, futures: Dict[str, Future]) -> None:
         self.render_sriov_node_policy(mgmtPolicyName, managementVFsAll, numVfs, managementResourceName, mgmtPolicyFile)
 
         print(client.oc("create -f manifests/nicmode/sriov-pool-config.yaml"))
+        self.wait_for_mcp(client, "sriov", "sriov-pool-config.yaml")
```

Comment:
Let's do this in the client class. This more robust wait can be used elsewhere too